langroid/agent/chat_agent.py

```diff
@@ -145,10 +172,22 @@ def __init__(
         self.system_tool_format_instructions: str = ""
 
         self.llm_functions_map: Dict[str, LLMFunctionSpec] = {}
+        self.llm_functions_class_map: Dict[str, type[ToolMessage]] = {}
```

Comment:
is this needed since we already have `self.llm_tools_map` (defined in `agent/base.py`) ? If it's needed, suggest adding short comment about it.